metalearn/metafeatures/base.py

```diff
@@ -28,6 +28,20 @@ def __init__(
             A custom map of ``computer``'s argument names to the global resource names
             that will be passed as ``computer``'s arguments when ``computer`` is called.
         """
+
+        computer_args = inspect.getfullargspec(computer)
+        # TODO: If needed, add support for `computer` functions that
+        # use these types of arguments.
```

Comment:
Currently, none of the resource-computing or metafeature-computing functions in the `metalearn` package use varargs, varkw, or keyword-only args, so this is not an issue, but it's good to have this check anyways since that behavior is not supported. I propose we leave this check here until the time comes when a resource-computing or metafeature-computing function _needs_ to use argument types other than positional args (if that time comes).